File: 1º Semestre/Programação de Computadores/Lista de Exercicios 2/10.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(){
	float nota1, nota2, nota3;
	
	printf("Digite a primeira nota: ");
	scanf("%f", &nota1);
	
	printf("Digite a segunda nota: ");
	scanf("%f", &nota2);
	
	printf("Digite a terceira nota: ");
	scanf("%f", &nota3);
	
	if(nota1 + nota2 + nota3 >= 6){
		printf("Voce foi aprovado! Sua media foi %0.2f pontos", nota1 + nota2 + nota3);
	} else {
		printf("Voce foi reprovado! Sua media foi %0.2f pontos", nota1 + nota2 + nota3);
	}
}

File: 1º Semestre/Programação de Computadores/Lista de Exercicios 2/Python/10.py
"""
10. O sistema de avaliação de determinada disciplina é composto por três provas. A primeira
prova tem peso 2, a segunda tem peso 3 e a terceira tem peso 5. Considerando que a
média para aprovação é 6.0, faça um algoritmo para calcular a média final de um aluno
desta disciplina e dizer se o aluno foi aprovado ou não.
"""

nota1 = float(input("Digite a primeira nota: "))
nota2 = float(input("Digite a segunda nota: "))
nota3 = float(input("Digite a terceira nota: "))
media = 6;

if(2 * nota1 + 3 * nota2 + 5 * nota3) / 10 < media:
    print("Você foi reprovou! Sua media foi de", (2 * nota1 + 3 * nota2 + 5 * nota3) / 10)
else:
    print("Você foi aprovou! Sua media foi de", (2 * nota1 + 3 * nota2 + 5 * nota3) / 10)